ribbon/env.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#----------------------------------------------------------------------------
+# Created By: Steven (steven@ribbon.finance)
+# Created Date: 04/04/2022
+# version ='0.1.0'
+# ---------------------------------------------------------------------------
+""" Module to load .env variables and JSON constants """
+# ---------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------
+# Imports
+# ---------------------------------------------------------------------------
+import os
+import json
+from dotenv import load_dotenv
+
+# ---------------------------------------------------------------------------
+# Constants
+# ---------------------------------------------------------------------------
+ADDRESSES_JSON = "./constants/addresses.json"
+CHAINS = ["kovan", "mainnet"]
```

Comment:
I'm thinking if we can combine the `dataclass` and `Enum` ([docs](https://docs.python.org/3/library/enum.html#creating-an-enum)) to have it documented in a better way.

I'll come up with a proposal.